### PR TITLE
[Clusterfile v2 ] modify build  module

### PR DESCRIPTION
**Pull request by kakaZhou719**

### CODE REVIEW

File: build/v2/buildkit/buildimage/build_image.go [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+	RegistryInfo      *buildinstruction.MountTarget
+}
+
+func (b BuildImage) ExecBuild(ctx Context) error {
+	var (
+		execCtx    buildinstruction.ExecContext
+		newLayers  = b.NewLayers
+		baseLayers = b.BaseLayers
+	)
+	if ctx.UseCache {
```
Comment (fanux) [inline build/v2/buildkit/buildimage/build_image.go:61]: Can we use cache by default?

Comment (kakaZhou719) [reply build/v2/buildkit/buildimage/build_image.go:61]: yes, UseCache default value is true. here we just transfer different situation。

File: build/buildkit/buildimage/registry.go [hunk 2]
```diff
@@ -87,13 +88,26 @@ func mountRootfs(res []string) (*buildinstruction.MountTarget, error) {
 	if err != nil {
 		return nil, err
 	}
-	err = startRegistry(rootfs.GetMountTarget())
-	if err != nil {
-		return nil, err
+	//if is sealer docker ,we need to start docker registry to collect docker image.some image in the crd operator, or copy offline image.
+	//if not ,no need to start docker registry. image name show in kuebfile layer :manifests,imageList ,charts.
+	if IsSealerDocker() {
```
Comment (fanux) [inline build/buildkit/buildimage/registry.go:93]: We still needs start the registry now? If using SaveImage?

File: docs/site/src/docs/advanced/develop-plugin.md [hunk 3]
```diff
@@ -16,6 +16,7 @@ if user doesn't want their plugin code to be open sourced, we can develop an out
 
 * package name must be "main"
 * exposed variable must be "Plugin"
```
Comment (fanux) [inline docs/site/src/docs/advanced/develop-plugin.md:18]: Plugin for what?

**Review by fanux — APPROVED** [review]

**Pull request merged**